website/server/models/user/schema.js

```diff
@@ -159,6 +159,7 @@ export const UserSchema = new Schema({
     duneBuddy: Boolean,
     roughRider: Boolean,
     rodentRuler: Boolean,
+    cats: Boolean,
```

Comment:
is there a particular reason this is just `cats` instead of the actual phrase in camelCase like most of the previous achievements in this style?

Reply:
The achievement image is names `achievement-cats2x.png`.